src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagEmailService.kt

```diff
@@ -76,14 +79,17 @@ class ProductaanvraagEmailService @Inject constructor(
     ) {
         mailTemplateService.findMailtemplateByName(automaticEmailConfirmation.templateName)?.let {
                 mailTemplate ->
+            val replyToAddress = automaticEmailConfirmation.emailReply?.let {
+                MailAdres(it.extractEmailAddress(configuratieService), null)
+            }
             val mailGegevens = MailGegevens(
-                MailAdres(automaticEmailConfirmation.emailSender, null),
-                MailAdres(to, null),
-                MailAdres(automaticEmailConfirmation.emailReply, null),
-                mailTemplate.onderwerp,
-                mailTemplate.body,
-                null,
-                true
+                from = MailAdres(automaticEmailConfirmation.emailSender.extractEmailAddress(configuratieService), null),
```

Comment:
Why do this inline but use a val for the `replyToAddress`? Please make this consistent. Either use a val for both or do it inline for both.